models/quickswap/polygon/quickswap_polygon_schema.yml

```diff
@@ -96,12 +96,61 @@ models:
         name: evt_index
         description: ""
 
+  - name: quickswap_v2_polygon_trades
+    meta:
+      blockchain: polygon
+      sector: dex
+      project: quickswap_v2
+      contributors: maybeYonas
+    config:
+      tags: ['polygon','quickswap_v2','trades', 'quickswap','dex']
+    description: >
+        Quickswap V2 contract trades on Polygon
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_seed:
+          blockchain: polygon
+          project: quickswap
+          version: 2
```

Comment:
Hi @darvinrio,
in order for `check_dex_seed` to work, you may need to add some seed records in this file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

Reply:
@Hosuke added the request seed records